### DISCUSSION TITLE
Bump supabase-js from v1 to v2

### DISCUSSION THREAD

**Discussion by dustin-we** [supabase/supabase#11980] [Feature Requests]
Hi,

would it be possible to bump the dependency of supabase-js from v1 to v2?

I am testing it today and will get back to you, but wanted to make sure with the creator first, if this is even an option?

Thanks!

**Comment by GaryAustin1** [comment]
Not super clear on what you are asking, but there are breaking changes between ver1 and ver2 that require code changes.

https://supabase.com/docs/reference/javascript/release-notes

**Reply by dustin-we** [reply]
I am sorry, I clicked on discussion on vue-supabase and somehow landed here?

I am deleting this

**Comment by dustin-we** [comment]
Looks like a did a mistake.

I wanted to start this discussion in supabase-community/vue-supabase, but somehow landed here.
I'll open a feature requests in that repo, feel free to delete this post.

Sorry!
